### ISSUE TITLE
Licence `total_managed_memory` computed using decimal GB but memory sizes typically use binary GiB

### ISSUE THREAD

**Issue by DaveCTurner** [elastic/cloud-on-k8s#5465]
A user reported confusion to me about the `total_managed_memory` field of the license:

> 259GB as total_managed_memory but as elastic nodes we have 7 x 30gb data nodes + 16 gb master node + 16 gb kibana = 242GB. So I don't understand where is the difference?

The explanation seems to be the difference between multiples-of-1024 Gibibytes and multiples-of-1000 Gigabytes: 242×1024×1024×1024 ≅ 259×1000×1000×1000. Typically we use binary gibibytes to describe memory sizes (e.g. the JVM argument `-Xmx16g` gives you a 16 GiB node which is a little over 17.1 GB in decimal) so I think the confusion is justified. Could we fix this or at least avoid the ambiguity somehow?

**Comment by pebrc** [comment]
We discussed this a bit out of band and agree that moving to GiB would probably be easier to understand and would  avoid the confusion. One thing we want to clarify upfront is which unit the Enterprise Resource Units are based on just to make sure everything aligns here eventually and we don't introduce a new source of confusion.

**Comment by thbkrkr** [comment]
ERUs are calculated using GB (multiples of 10).

**Comment by pebrc** [comment]
> ERUs are calculated using GB (multiples of 10).

That is unfortunate. But as all other user interfaces including in ECE and Elastic Cloud return GiB (often incorrectly labeled as GB) I think we should still do the same and return GiB despite this inconsistency.

**Comment by DaveCTurner** [comment]
FWIW Elasticsearch APIs typically avoid this ambiguity by reporting both the raw number of bytes and a "human-readable" version of the same number. I find the redundancy often helpful, although you can switch it off with `?human=false` if the parsimony is more important.

I'd also rather see this number in GiB, but I can see the arguments in both directions. Either way, if the raw value were also available then there would be much less confusion.

**Comment by lucianaparaschivei** [comment]
Basically I think for an end user is important to know how much license it is using vs how much he needs to buy. We have multiple elastic clusters running with x amount of Enterprise Resource Units. So this difference can be that large that we need to decide if we need an extra resource unit or not based on GiB vs GB which is an extra cost.

**Comment by thbkrkr** [comment]
We will report the total managed memory in GiB and raw bytes to avoid confusion.

Current:
```
{
  ...
  "total_managed_memory": "68.72GB"
}
```

Proposed change:
```
{
   ...
  "total_managed_memory_bytes": "68719476736",
  "total_managed_memory": "64.00GiB"
}
```

**Comment by lucianaparaschivei** [comment]
> We will report the total managed memory in GiB and raw bytes to avoid confusion.
> 
> Current:
> 
> ```
> {
>   ...
>   "total_managed_memory": "68.72GB"
> }
> ```
> 
> Proposed change:
> 
> ```
> {
>    ...
>   "total_managed_memory_bytes": "68719476736",
>   "total_managed_memory": "64.00GiB"
> }
> ```

In this case 1 ERU equals 64.00GiB equals 68.7GB ??

**Comment by thbkrkr** [comment]
> In this case 1 ERU equals 64.00GiB equals 68.7GB ??

Yes, ERUs are the total of bytes divided by 64.00GiB (in the above example: `68719476736 / (64*1024*1024*1024) = 1`)
